include/SFML/Window/Keyboard.hpp

```diff
@@ -153,6 +158,158 @@ class SFML_WINDOW_API Keyboard
+    /// on the keyboard layout used by the operating system. Usually, the AT-101
+    /// keyboard can be used as reference for the physical position of the keys.
+    ///
+    /// The scancodes are based on a subset of Table 12: Keyboard/Keypad Page
+    /// of Universal Serial Bus (USB): HID Usage Tables, v1.12.
+    ///
+    /// \todo When porting this for SFML 3, remove the `s` prefix and use
+    ///       enum class.
+    ///
+    ////////////////////////////////////////////////////////////
+    enum Scancode
+    {
+        ScanUnknown = -1,      ///< Represents any scancode not present in this enum
+        ScanA = 0,             ///< Keyboard a and A key
+        ScanB,                 ///< Keyboard b and B key
+        ScanC,                 ///< Keyboard c and C key
+        ScanD,                 ///< Keyboard d and D key
+        ScanE,                 ///< Keyboard e and E key
+        ScanF,                 ///< Keyboard f and F key
+        ScanG,                 ///< Keyboard g and G key
+        ScanH,                 ///< Keyboard h and H key
+        ScanI,                 ///< Keyboard i and I key
+        ScanJ,                 ///< Keyboard j and J key
+        ScanK,                 ///< Keyboard k and K key
+        ScanL,                 ///< Keyboard l and L key
+        ScanM,                 ///< Keyboard m and M key
+        ScanN,                 ///< Keyboard n and N key
+        ScanO,                 ///< Keyboard o and O key
+        ScanP,                 ///< Keyboard p and P key
+        ScanQ,                 ///< Keyboard q and Q key
+        ScanR,                 ///< Keyboard r and R key
+        ScanS,                 ///< Keyboard s and S key
+        ScanT,                 ///< Keyboard t and T key
+        ScanU,                 ///< Keyboard u and U key
+        ScanV,                 ///< Keyboard v and V key
+        ScanW,                 ///< Keyboard w and W key
+        ScanX,                 ///< Keyboard x and X key
+        ScanY,                 ///< Keyboard y and Y key
+        ScanZ,                 ///< Keyboard z and Z key
+        ScanNum1,              ///< Keyboard 1 and ! key
+        ScanNum2,              ///< Keyboard 2 and @ key
+        ScanNum3,              ///< Keyboard 3 and # key
+        ScanNum4,              ///< Keyboard 4 and $ key
+        ScanNum5,              ///< Keyboard 5 and % key
+        ScanNum6,              ///< Keyboard 6 and ^ key
+        ScanNum7,              ///< Keyboard 7 and & key
+        ScanNum8,              ///< Keyboard 8 and * key
+        ScanNum9,              ///< Keyboard 9 and ) key
+        ScanNum0,              ///< Keyboard 0 and ) key
+        ScanEnter,             ///< Keyboard Return (ENTER) key
+        ScanEscape,            ///< Keyboard Escape key
+        ScanBackspace,         ///< Keyboard Backspace key
+        // TODO above it's BackSpace, but is it correct? What do we use here?
```

Comment:
What do we want? 

 1. A consistent typo and use `ScalBackSpace`, or
 2. Deprecate `BackSpace`, introduce `Backspace`, keep `ScanBackspace`?

Reply:
According to wikipedia it's a single word, so Backspace.

Reply:
Okay, I'll create an independant PR that introduces `Backspace` and deprecates `BackSpace` (and for the backslash code as well).

Reply:
`ScanBackspace`